### PR TITLE
Upgrade phpstan

### PR DESCRIPTION
**Pull request by carakas**
## Type

- Non critical bugfix
- Enhancement

## Resolves the following issues

Fixes #3210

## Pull request description

Upgrade phpstan to the latest version and make it pass on level 1

### CODE REVIEW

File: src/Backend/Modules/FormBuilder/Ajax/SaveField.php [hunk 1]
```diff
@@ -264,7 +264,7 @@ public function execute(): void
         if ($label !== '') {
             $settings['label'] = \SpoonFilter::htmlspecialchars($label);
         }
-        if (isset($values)) {
+        if (count($values) > 0) {
```
Comment (jessedobbelaere) [inline src/Backend/Modules/FormBuilder/Ajax/SaveField.php:267]: I see an if check for  `$values !== ''` somewhere above. Doing a count on a non-array/non-countable will fail from PHP >7.2 with `count(): Parameter must be an array or an object that implements Countable`

If we are not sure its an array or countable, we could use `if (!empty($values)) {` ?

File: src/Backend/Modules/MediaLibrary/Domain/MediaFolder/MediaFolder.php [hunk 2]
```diff
@@ -82,7 +82,7 @@ class MediaFolder implements JsonSerializable
     protected $items;
 
     /**
-     * @var ArrayCollection
+     * @var Collection<MediaFolder>
```
Comment (jessedobbelaere) [inline src/Backend/Modules/MediaLibrary/Domain/MediaFolder/MediaFolder.php:85]: 👌

**Review by jessedobbelaere — APPROVED** [review]
Awesome job!! 🥇

File: src/Backend/Modules/FormBuilder/Ajax/SaveField.php (hunk 1 above)
Comment (carakas) [reply src/Backend/Modules/FormBuilder/Ajax/SaveField.php:267]: Good catch, I'll check if I can make it fail :p

**Pull request merged**